package-lock.json

```diff
@@ -423,6 +423,11 @@
         "is-obj": "^2.0.0"
       }
     },
+    "dotenv": {
```

Comment:
I Tried this too but couldn't do it properly

Reply:
you need a .env file